### ISSUE TITLE
How to comment multiple lines on German (QWERTZ) keyboard

### ISSUE THREAD

**Issue by qebrahym** [jupyterlab/jupyterlab#13128]
Hello everyone, 
i have tried and tested almost every possible method and solution from internet and suggested by different people here but still got no success. 

Can someone pls help me out **"How can i COMMENT OUT MULTIPLE LINES in JUPYTER NOTEBOOK" on Mac** with German Keyboard layout. 

Your +ve and kind response will be highly appreciated. 
Thanks

**Comment by firai** [comment]
Have you tried `Cmd + ß` (from https://github.com/jupyterlab/jupyterlab/issues/4778#issuecomment-944970341)?

**Comment by qebrahym** [comment]
> Have you tried `Cmd + ß` (from [#4778 (comment)](https://github.com/jupyterlab/jupyterlab/issues/4778#issuecomment-944970341))?

Yes I tried, but it doesn’t work.

**Comment by jweill-aws** [comment]
The key combination using a US English keyboard is <kbd>CMD+/</kbd> but it looks like <kbd>/</kbd> (forward slash) is a shifted character on a QWERTZ keyboard.

**Comment by AlexanderMelde** [comment]
As a general suggestion, the <kbd>/</kbd> key on the numpad usually works, but I'm not sure if that helps you in this particular case.

**Comment by qebrahym** [comment]
> The key combination using a US English keyboard is CMD+/ but it looks like / (forward slash) is a shifted character on a QWERTZ keyboard.

Yes indeed that‘s the issue. the Slash / button on QWERTY Keyboard doesn‘t work.

**Comment by qebrahym** [comment]
> As a general suggestion, the / key on the numpad usually works, but I'm not sure if that helps you in this particular case.

Actually i want to use Slash / button on QWERTY Keyboard to comment out multiple lines. Because the Keyboard Layout changes and it get difficult to type quickly on US keyboard as all the buttons are different and more importantly it affects the typing speed and impact the workflow.

**Comment by MatusGasparik** [comment]
This is also my pain point with jupyterlab. I still haven't figured out how to comment out stuff in the notebook cells, other than manually. There is the multi-line cursor (with Alt-key), but I'd rather not use mouse. No even the `jupyterlab-vim` extension is of any help (unless it is possible to add custom plugins, like `vim-commentary`, which I don't know if its possible...).

Any working solutions will be very appreciated!

**Comment by v18nguye** [comment]
Let's try with 
`Strg  + divide button`

**Comment by MatusGasparik** [comment]
Thanks, @v18nguye. But, that works (with the German keyboard at least) only with the „/„ on the number pad. If your keyboard doesn‘t have one, no luck.

**Comment by lenascherer** [comment]
Hi, not sure if you have found a solution yet, as the feed is already a year old, but for me cmd + SHIFT + 7 worked well on the German keyboard.

**Comment by saurabh-2905** [comment]
> Hi, not sure if you have found a solution yet, as the feed is already a year old, but for me cmd + SHIFT + 7 worked well on the German keyboard.

which browser are you using? For me, it opens the help menu on chrome.

**Comment by lenascherer** [comment]
Ah, good point. I am using Visual Studio Code and the Jupyter Notebook extension there. At least for me, it is much more convenient than using Jupyter Notebook in the browser.

**Comment by saurabh-2905** [comment]
Yes, it makes sense. Let me try switching as well. Thanks.

**Comment by rinlinh00** [comment]
For me the combination Ctrl+# on German keyboard layout works like a charm with every programming language in VSCode

**Comment by fedor98** [comment]
hey, 

don't know what program you are using. But I had the same problem with colab today.

if you have a mac you can try: ´command´ + ´control´ + ´shift´ + ´7´ 

I think there will be something similiar for windows

**Comment by fedor98** [comment]
hey,

don't know what program you are using. But I had the same problem with colab today.

if you have a mac you can try: "command"+ "control" + "shift" + "7"

I think there will be something similiar for windows

**Comment by saurabh-2905** [comment]
Yes, I am using jupyter in VSCode now, and there I can use the key bindings that I have configured. Since I have configured "command"+"#" for comment toggle, it works fine. The initial shortcut was "command"+ "control" + "shift" + "7",  but I was way irritating.

**Comment by nancymuyeh** [comment]
> The key combination using a US English keyboard is CMD+/ but it looks like / (forward slash) is a shifted character on a QWERTZ keyboard.

I used Cmd+Shift+7 since the second function for 7 is / and it works

**Comment by Linchenpal** [comment]
did someone solve it? im having the same issue, cmd+shift+/  isnt working instead helps menu is being open...(chrome)

**Comment by anisoooo** [comment]
> did someone solve it? im having the same issue, cmd+shift+/ isnt working instead helps menu is being open...(chrome)

cmd + k + c    to put it as a comment  
cmd + k + u    to uncomment it        
i use mac with german keyboard :)
